po/be.po

```diff
@@ -355,104 +389,146 @@ msgid "Allow applications to inhibit system shutdown"
 
 #: src/login/org.freedesktop.login1.policy:33
 msgid "Allow applications to delay system shutdown"
 msgstr "Дазволіць праграмам затрымліваць выключэнне сістэмы"
 
 #: src/login/org.freedesktop.login1.policy:34
 msgid "Authentication is required for an application to delay system shutdown."
-msgstr "Для таго, каб дазволіць праграмам затрымліваць выключэнне сістэмы, патрабуецца аўтэнтыфікацыя."
+msgstr ""
+"Для таго, каб дазволіць праграмам затрымліваць выключэнне сістэмы, "
+"патрабуецца аўтэнтыфікацыя."
 
 #: src/login/org.freedesktop.login1.policy:44
 msgid "Allow applications to inhibit system sleep"
 msgstr "Дазволіць праграмам перашкаджаць пераходу сістэмы ў рэжым сну"
 
 #: src/login/org.freedesktop.login1.policy:45
 msgid "Authentication is required for an application to inhibit system sleep."
-msgstr "Для таго, каб дазволіць праграмам перашкаджаць пераходу сістэмы ў рэжым сну, патрабуецца аўтэнтыфікацыя."
+msgstr ""
+"Для таго, каб дазволіць праграмам перашкаджаць пераходу сістэмы ў рэжым сну, "
+"патрабуецца аўтэнтыфікацыя."
 
 #: src/login/org.freedesktop.login1.policy:55
 msgid "Allow applications to delay system sleep"
 msgstr "Дазволіць праграмам затрымліваць пераход сістэмы ў рэжым сну"
 
 #: src/login/org.freedesktop.login1.policy:56
 msgid "Authentication is required for an application to delay system sleep."
-msgstr "Для таго, каб дазволіць праграмам затрымліваць пераход сістэмы ў рэжым сну, патрабуецца аўтэнтыфікацыя."
+msgstr ""
+"Для таго, каб дазволіць праграмам затрымліваць пераход сістэмы ў рэжым сну, "
+"патрабуецца аўтэнтыфікацыя."
 
 #: src/login/org.freedesktop.login1.policy:65
 msgid "Allow applications to inhibit automatic system suspend"
 msgstr "Дазволіць праграмам перашкаджаць аўтаматычнаму прыпыненню сістэмы"
 
 #: src/login/org.freedesktop.login1.policy:66
-msgid "Authentication is required for an application to inhibit automatic system suspend."
-msgstr "Для таго, каб дазволіць праграмам перашкаджаць аўтаматычнаму прыпыненню сістэмы, патрабуецца аўтэнтыфікацыя."
+msgid ""
+"Authentication is required for an application to inhibit automatic system "
+"suspend."
+msgstr ""
+"Для таго, каб дазволіць праграмам перашкаджаць аўтаматычнаму прыпыненню "
+"сістэмы, патрабуецца аўтэнтыфікацыя."
 
 #: src/login/org.freedesktop.login1.policy:75
 msgid "Allow applications to inhibit system handling of the power key"
 msgstr "Дазволіць праграмам перашкаджаць сістэме апрацоўваць кнопку выключэння"
 
 #: src/login/org.freedesktop.login1.policy:76
-msgid "Authentication is required for an application to inhibit system handling of the power key."
-msgstr "Для таго, каб дазволіць праграмам перашкаджаць сістэме апрацоўваць кнопку выключэння, патрабуецца аўтэнтыфікацыя."
+msgid ""
+"Authentication is required for an application to inhibit system handling of "
+"the power key."
+msgstr ""
+"Для таго, каб дазволіць праграмам перашкаджаць сістэме апрацоўваць кнопку "
+"выключэння, патрабуецца аўтэнтыфікацыя."
```

Comment:
The changes in this the .po files seem to be mostly trivial indentation changes because of a different editor.  I'd prefer to skip those.

Reply:
weblate sent the same ones, so I think something might have changed recently? anyway, dropped